lib/blocktree/blocktree.go

```diff
@@ -297,29 +298,51 @@ func (bt *BlockTree) Leaves() []Hash {
 	return la
 }
 
-// HighestCommonAncestor returns the highest block that is a Ancestor to both a and b
-func (bt *BlockTree) HighestCommonAncestor(a, b Hash) (Hash, error) {
-	bt.RLock()
-	defer bt.RUnlock()
```

Comment:
is it ok we don't lock the block tree anymore? :thinking:

Reply:
Yeah I think so too, good catch